xdsl/dialects/builtin.py

```diff
@@ -371,6 +428,7 @@ def verify(self) -> None:
                     f"element is of type {type(val)}")
 
     @staticmethod
+    @deprecated("__init__ should be used instead")
```

Comment:
Can we generate these error messages automatically?

So that they maybe print `calls to <funcname> are deprecated, please use constructors instead`?

Reply:
I think the decorator already gives you the info which function was called, if you actually run into it.

Reply:
Changed it by adding a new decorator to give the second part of the message directly.